### ISSUE TITLE
Pen touch input

### ISSUE THREAD

**Issue by vsboost** [cjcliffe/CubicSDR#208]
Hi Charles,
Not sure if its CubicSDR or something else but using Ubuntu on my surface pro CubicSDR wont allow me to select anywhere on the waterfall using the pen, it works on other parts of the program like changing frequencies / selecting menu items or modulation types, i have to use my finger to make a selection on the waterfall.

Hope that makes sense.

Cheers

**Comment by cjcliffe** [comment]
@vsboost I'm guessing it's almost impossible to lift the pen without moving a few pixels -- CubicSDR's tolerance for a drag vs. a tap is very small -- I may have to tweak it a bit to allow a bit of movement to still register as a click instead of drag..

**Comment by vsboost** [comment]
You are correct, if i am ultra stable with the pen when i click it works as it should.
